### PR TITLE
Fix CouchDB management commands for version >= 3

### PR DESCRIPTION
**Pull request by skodde**
##### ENVIRONMENTS AFFECTED
All

### CODE REVIEW

File: src/commcare_cloud/commands/ansible/ops_tool.py [hunk 1]
```diff
@@ -254,11 +255,15 @@ class CouchDBClusterInfo(CommandBase):
         Argument("--database", help="Only show output for this database"),
         Argument("--couch-port", default=15984, type=int, help="CouchDB port. Defaults to 15984"),
         Argument("--couch-local-port", default=15986, type=int, help="CouchDB node local port. Defaults to 15986"),
```
Comment (snopoke) [inline src/commcare_cloud/commands/ansible/ops_tool.py:257]:
```suggestion
        Argument("--couch-local-port", default=15986, type=int, help="CouchDB node local port. Only applicable to CouchDB version < 3.0. Defaults to 15986"),
```

**Review by snopoke — APPROVED** [review]
Small suggestions but otherwise looks ok.

**Comment by dannyroberts** [comment]
@skodde it looks like you have a few options to deal with the build errors like `FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/dimagi/commcare-cloud/tests/couch_migration_config/env1/public.yml'`.

1. Add a `public.yml` containing just `{}` to all the test envs under `tests/test_envs/`
2. Make `environment.public_vars` return `{}` if the file isn't found
3. Handle `FileNotFoundError` whenever you call `environment.public_vars` (I dislike this one the most.)

File: src/commcare_cloud/commands/ansible/ops_tool.py (hunk 1 above)
Comment (skodde) [reply src/commcare_cloud/commands/ansible/ops_tool.py]: Change integrated. Thanks!

**Comment by skodde** [comment]
> 2. Make `environment.public_vars` return `{}` if the file isn't found

:+1:

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**